### PR TITLE
add gradle support

### PR DESCRIPTION
**Pull request by brumple**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+plugins {
+    id 'java'
+    id 'org.jetbrains.intellij' version '0.4.8'
+}
+
+group 'com.slacklane'
```
Comment (rdewell) [inline build.gradle]: Should be com.stacklane "t" not "l"

Comment (brumple) [reply build.gradle]: I thought that I had committed that to memory because initially I was really messing that up :)

**Comment by brumple** [comment]
There are a couple options to build the plugin:

1. The original Stacklane.iml should still work
2. gradle cli
a) ./gradlew tasks - shows all the available build targets
b) ./gradlew buildPlugin - Bundles the project as a distribution.
c) ./gradlew runIde - Runs Intellij IDEA with installed plugin.  This launches a clean IDE with the plugin. Not should it this is helpful for targeting WebStorm, but worth playing with
3. IntelliJ gradle run configuration

I believe that IntelliJ will be doing the normal build using the gradle setup.  Then probably just need to run the buildPlugin gradle task either on the cli or run config.  I haven't used gradle on IntelliJ so there might be other options that I'm not aware of.

**Comment by rdewell** [comment]
OK, build make sense.  

I'm seeing somethings in IntelliJ that I can't easily resolve.  

1.

content root

2. Do you have any idea how you have "IntelliJ IDEA IU-183.xyz..."?  I'm using 2019.1.2, but that's only giving me an option for "IntelliJ IDEA IU-181.xyz...".   

3.  Could be related to above, but when I open resources/META-INF/plugin.xml, it's reporting that it can find the configurationType or runConfigurationProducer.

**Pull request closed without merging**